src/lib/routing-table.js

```diff
@@ -35,6 +35,28 @@ class RoutingTable {
     return false
   }
 
+  /**
+   * Compute a curve quote's `appliesToPrefix`, which is the shortest prefix
+   * that uniquely matches the target.
+   *
+   * @param {IlpAddress} routePrefix
+   * @param {IlpAddress} destinationAccount
+   * @returns {IlpAddress}
+   */
+  getAppliesToPrefix (routePrefix, destinationAccount) {
```

Comment:
if the route already has targetPrefix "" but does not have a curve, and the liquidity quote gives you that missing curve, then you should add it, right? that peer is telling you that you can use this curve for any address under ""

it will not affect your `abc.` route, because you have the more specific route for that which uses a different `nextLedger`, and therefore also has its own curve.

what it does tell you is that the two routes have different values for `nextLedger`. A route without a curve can only have one `nextLedger`, for routes with a curve, the `nextLedger` depends on the point on the curve.